sound/hybrid_font.h

```diff
@@ -369,16 +374,23 @@ class HybridFont : public SaberBase {
   bool swinging_ = false;
   void SB_Motion(const Vec3& gyro, bool clear) override {
     float speed = sqrtf(gyro.z * gyro.z + gyro.y * gyro.y);
-    if (speed > config_.ProffieOSSwingSpeedThreshold) {
-      if (!swinging_ && state_ != STATE_OFF &&
+    uint32_t now = micros();
+    uint32_t delta_micros = now - last_swing_micros_;
+    last_swing_micros_ = now;
+    float delta = delta_micros * 0.000001;
+    float current_acceleration = (speed - last_speed_) / delta;
+    last_speed_ = speed;
+    float filter_factor = powf(0.2, delta);
+    swing_acceleration_ = swing_acceleration_ * filter_factor + current_acceleration * (1 - filter_factor);
+    if (speed > config_.ProffieOSSwingSpeedThreshold && state_ != STATE_OFF &&
 	  !(lockup.files_found() && SaberBase::Lockup())) {
-        swinging_ = true;
-        StartSwing();
-      }
+      StartSwing(swing_acceleration_);
+      swinging_ = true;
       float swing_strength = std::min<float>(1.0, speed / config_.ProffieOSSwingSpeedThreshold);
-      SetSwingVolume(swing_strength, 1.0);
-    } else if (swinging_ && speed <= config_.ProffieOSSwingSpeedThreshold) {
+      //SetSwingVolume(swing_strength, 1.0);
```

Comment:
Why is this commented out now?

Reply:
I was getting terrible crackles and pops on slashes that overlap heavily, just commented out during testing until I could get back to that issue and see what the issue is.

Reply:
reverted